docs/user-docs/latest/media-library/organizing-assets-with-folders.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Adding assets to the Media Library - Strapi User Guide
+description:
+canonicalUrl:
+---
+
+<!-- TODO: update SEO -->
+
+# Organizing assets with folders
+
+Folders in the [Media Library](/user-docs/latest/media-library/introduction-to-media-library.md) help you organize uploaded assets. Folders sit at the top of the Media Library view or are accessible from the Media field popup when using the Content Manager.
```

Comment:
Not sure the link here is necessary 🤔  However we could add another link to the Content Manager documentation.